app/javascript/components/concept-map/ConceptMap.css

```diff
@@ -69,32 +69,25 @@
                 }
 
                 & .display {
-                    @apply pt-12 pb-8 px-24 flex items-center justify-between;
+                    @apply py-12 px-24 flex items-center justify-between;
 
                     & .name {
                         @apply text-16 font-medium leading-code;
                     }
 
-                    & .stats {
-                    }
-
-                    & .status {
-                    }
-
                     & .complete-icon {
-                        @apply ml-8;
+                        @apply ml-12;
+                        width: 16px;
+                        height: 16px;
                     }
                 }
 
                 & progress {
-                    -webkit-appearance: none;
-                    appearance: none;
-
-                    width: 100%;
+                    @apply appearance-none w-100;
                     height: 4px;
 
                     &.hidden {
-                        visibility: hidden;
+                        @apply hidden;
```

Comment:
This is `display:none` (confusingly named by tailwind!)

Reply:
I deviated here on purpose to maintain flex-box alignment

Reply:
Cool. Also fixed by the latest commit 🙂

Reply:
I think it's because they use `invisible`. Good thing that `[hidden]` should be `@apply hidden`, and thus the naming is again okay 👍